Test/Unit/Helper/GraphQL/ResponseObserver.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Test\Unit\Helper\GraphQL;
+
+
+class ResponseObserver {
```

Comment:
you can use Bolt\Boltpay\Model\Response instead

Reply:
Yes but if I use response I am unable to store the the  response

Reply:
As in I don't know how to test it

Reply:
nvm got it